base/loading.jl

```diff
@@ -407,9 +407,24 @@ end
 
 ## generic project & manifest API ##
 
-const project_names = ("JuliaProject.toml", "Project.toml")
-const manifest_names = ("JuliaManifest.toml", "Manifest.toml")
-const preferences_names = ("JuliaLocalPreferences.toml", "LocalPreferences.toml")
+const project_names = ["JuliaProject.toml", "Project.toml"]
+const default_manifest_names = ["JuliaManifest.toml", "Manifest.toml"]
```

Comment:
I think this could be populated on startup instead, and still be a global constant called `manifest_names`. I would think that would help tooling which uses this too.